webapp/app/perf.html

```diff
@@ -28,7 +30,6 @@
   <script src="vendor/jquery.flot.js"></script>
   <script src="vendor/jquery.flot.time.js"></script>
   <script src="vendor/mousetrap.min.js"></script>
-  <script src="js/config/local.conf.js"></script>
```

Comment:
Uh oops. Sunlight on screen + viewing angle slightly off meant I couldn't see the green. Ignore me :-)